cli_tools/common/distro/distro.go

```diff
@@ -167,6 +206,44 @@ func newCommonLinuxRelease(distro string, major, minor int) (Release, error) {
 	}, nil
 }
 
+// windowsRelease uses marketing versions rather than NT versions.
+// Currently the only minor version is "r2". For example, the versions
+// 2012 and 2012r2 and *not* import compatible, and have different
+// minor versions. In the first case, the minor version is empty.
+// In the second it is "r2".
+type windowsRelease struct {
+	major, minor, architecture string
+}
+
+func (w windowsRelease) ImportCompatible(other Release) bool {
+	actualOther, ok := other.(windowsRelease)
+	compat := ok &&
+		w.major == actualOther.major &&
+		w.architecture == actualOther.architecture
+	if w.minor == "r2" || actualOther.minor == "r2" {
+		compat = compat && (w.minor == actualOther.minor)
+	}
+	return compat
+}
+
+func (w windowsRelease) AsGcloudArg() string {
+	arg := fmt.Sprintf("windows-%s", w.major)
+	if w.minor == "r2" {
```

Comment:
Should it throw fatal if it's not "r2"?

Reply:
There could be other minor versions (eg: 8 vs 8.1), but "r2" is the only minor version that we treat separately. Added a comment to describe this.